src/main/windows/windowManager.ts

```diff
@@ -203,8 +203,12 @@ export class WindowManager {
     }
 
     handleResizedMainWindow = () => {
+        log.silly('WindowManager.handleResizedMainWindow');
+
         if (this.mainWindow) {
-            this.throttledWillResize(this.mainWindow?.getContentBounds());
+            const bounds = this.getBounds();
```

Comment:
This will never need to be different, since the `resized` event is only emitted on Windows/macOS. See here: https://www.electronjs.org/docs/latest/api/browser-window#event-resized-macos-windows

Reply:
Got it!
Would still be ok to have `getBounds` as the source of truth for bounds within the window manager?

Reply:
I think it's fine, just means an extra hoop for Linux to jump through but it's not a massive performance concern.